src/models/MyAccount.ts

```diff
@@ -168,8 +173,12 @@ export default class MyAccount {
       expirationDate: patron.expirationDate,
       primaryEmail: patron.emails.length > 0 ? patron.emails[0] : "",
       emails: patron.emails,
-      primaryPhone: patron.phones.length > 0 ? patron.phones[0].number : "",
-      phones: patron.phones,
+      primaryPhone: patron.phones
```

Comment:
I feel like this syntax can be simplified a bit to avoid the double ternary, maybe something like:

patron?.phones[0]?.number || ""

or

patron?.phones?.length ? patron.phones[0].number || ""